packages/cli/src/upload/upload.js

```diff
@@ -342,20 +342,33 @@ async function getPreviousUrlMap(options) {
  * @return {Promise<void>}
  */
 async function writeUrlMapToApi(urlMap) {
+  print(`Saving URL map to temporary public storage...\n`);
   const slug = getGitHubRepoSlug();
-  if (!slug) return;
+
+  if (!slug) {
+    print(`No GitHub repository slug found, skipping.\n`);
+    return;
+  }
+  print(`Saving URL map under GitHub repository slug ${slug}...`);
 
   try {
     /** @type {Record<string, string>} */
     const payload = {slug};
     Array.from(urlMap.entries()).forEach(([k, v]) => (payload[k] = v));
-    await fetch(SAVE_URL_MAP_URL, {
+
+    const response = await fetch(SAVE_URL_MAP_URL, {
       method: 'POST',
       body: JSON.stringify(payload),
       headers: {'content-type': 'application/json'},
     });
+
+    if (response.ok) {
+      print(`success!\n`);
+    } else {
+      print(`failed!\n`);
+    }
   } catch (err) {
-    print(`Failed to save urlMap: ${err.message}`);
+    print(`Failed to save URL map: ${err.message}`);
```

Comment:
```suggestion
    print(`Failed to save URL map: ${err.message}\n`);
```

missed this last time thanks for the fix!